plans/ci-1721-windows-pr-gate.md

```
@@ -0,0 +1,93 @@
+**2 ジョブに分けて `if:` でスキップする形も取らなかった。** スキップされたジョブは branch
+protection に対して成功として報告される — が、それは**プラットフォーム側の規則**であって、
+このリポジトリから確かめられるものではない。実際に成功する 1 ジョブなら、その規則が真である
+必要が無い。
+
+### 何を「Windows に見えない変更」とみなすか
+
+**当初 issue では「`server/infra/**` `server/session/**` `test/server/**` を触る PR だけ」を
+提案したが、調べたら成り立たない。** プラットフォームで分岐する spec は 9 ディレクトリに散っていて、
+`test/src/components/` もその 1 つ（#1719 がそこ）。
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)